cmd/util/app.go

```diff
@@ -89,6 +89,7 @@ type AppOptions struct {
 	retryBackoffMaxDuration         time.Duration
 	retryBackoffFactor              int64
 	ref                             string
+	SourceName                      string
```

Comment:
All other properties are lowercase.
```suggestion
	sourceName                      string
```

Reply:
@agaudreault Thank you for the review. Other fields in this struct, such as `Validate` and `Parameters`, are capitalized because they are used in the `commands` package. The same applies to the newly added `SourceName`.